contents/docs/self-host/deploy/upgrade-notes.md

```diff
@@ -132,3 +132,25 @@ If you didn’t make any customization to those, there’s nothing you need to d
     - `clickhouse.persistentVolumeClaim` -> `clickhouse.persistence.existingClaim`
 
     If you are overriding any of those, please make the corresponding changes before upgrading. Depending on your settings and setup, during this upgrade the ClickHouse pod might get recreated.
+
+### Upgrading from 10.x.x
+
+11.0.0 removes some legacy Helm annotations that are not necessary anymore (see [#179](https://github.com/PostHog/charts-clickhouse/pull/179) for more info).
+
+Before running the Helm upgrade, please run the following script first (note: replace the `RELEASE_NAME` and `RELEASE_NAMESPACE` accordingly if you are using a custom release name/namespace):
+
+```
+#!/usr/bin/env sh
+
+RELEASE_NAME="posthog"
+RELEASE_NAMESPACE="posthog"
+
+for deployment in $(kubectl -n $RELEASE_NAMESPACE get deployments --no-headers -o custom-columns=NAME:.metadata.name | grep "posthog")
```

Comment:
And being pedantic, should we also wrap `$RELEASE_NAMESPACE` in quotes everywhere

Reply:
In this case I don't think it is necessary (I've just passed this script on https://www.shellcheck.net/ with no issues) as it's a controlled input but we can do that anyway